static/beta/stage/navigation/settings-navigation.json

```diff
@@ -45,6 +45,12 @@
                 }
             ]
         },
+        {
+            "id": "idmsvc",
+            "appId": "idmsvc",
+            "title": "Directory & Domain Services",
+            "href": "/settings/idmsvc"
```

Comment:
Can you add a short description to the link? We require descriptions for links that are exposed in the services dropdown and by extension in the search results.

Reply:
@Hyperkid123 sure! thank you mate! done

I have rebased and squashed the commits.